src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --secondarycolor: #00acdc;
+  --primarycolor: #000;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 10px 0;
+  color: var(--primarycolor);
+}
+
+.nav__link:hover {
+  color: var(--secondarycolor);
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  text-transform: uppercase;
+  padding-left: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  color: var(--primarycolor);
+  position: relative;
+}
+
+[data-qa="hover"].nav__link {
+  color: var(--secondarycolor);
+}
+
+.nav__list:last-child {
+  margin-right: 50px;
```

Comment:
Get rid of this. Same for margin left for logo. Instead give padding of 50px to the parent element of both logo and nav